maven-release-plugin/src/main/java/org/apache/maven/plugins/release/PrepareReleaseMojo.java

```diff
@@ -101,6 +101,12 @@
     @Parameter( defaultValue = "true", property = "addSchema" )
     private boolean addSchema;
 
+    /**
+     * Comma separated profiles to enable on prepare, in addition to active profiles for project execution.
+     */
+    @Parameter( property = "preparationProfiles" )
+    private String preparationProfiles;
```

Comment:
`prepareProfiles` to match the mojo (goal) name.

Reply:
I chose this to match the `private String preparationGoals;`

Reply:
Granted.